projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -1,53 +1,74 @@
 import { Injectable } from '@angular/core';
 import { isNil } from 'lodash-es';
-import { Observable } from 'rxjs';
+import { combineLatest, Observable } from 'rxjs';
 import { map, shareReplay, take } from 'rxjs/operators';
+import { ApplicationFeature } from '../constants/application-constants';
+import { FeatureStateResolver } from '../feature/state/feature-state.resolver';
+import { FeatureState } from '../feature/state/feature.state';
 import { NavigationService } from '../navigation/navigation.service';
 import { PreferenceService, StorageType } from '../preference/preference.service';
+import { RelativeTimeRange } from './relative-time-range';
+import { TimeDuration } from './time-duration';
 import { TimeRange } from './time-range';
 import { TimeRangeService } from './time-range.service';
+import { TimeUnit } from './time-unit.type';
 
 @Injectable({ providedIn: 'root' })
 export class PageTimeRangePreferenceService {
   private static readonly STORAGE_TYPE: StorageType = StorageType.Local;
   private static readonly TIME_RANGE_PREFERENCE_KEY: string = 'page-time-range';
 
+  private readonly globalDefaultTimeRange: TimeRange = new RelativeTimeRange(new TimeDuration(30, TimeUnit.Minute));
```

Comment:
This should still be what it was before, one hour.

Reply:
Also given we can potentially assign it multiple times, we should construct it via a function, rather than assigning it as a constant.  (the relative discussion just had elsewhere)